src/select-list-view.ts

```diff
@@ -453,7 +485,7 @@ class ListItemView {
     this.domEventsDisposable.dispose()
   }
 
-  update (props) {
+  update (props: { element: EtchElement ; selected: boolean; onclick: () => void }) {
```

Comment:
Oh yeah since we're using the same props type twice, definitely make it its own type.

Reply:
Fixed in https://github.com/atom/atom-select-list/pull/28/commits/5abb4f0ef1e1a20ea56a4fa531da48de137718aa